db-migrations/33_update_multiple_deployments.up.sql

```diff
@@ -0,0 +1,31 @@
+-- For terminated endpoint, update the status of all successful deployments to 'terminated'
```

Comment:
For both queries below, were you able to test the performance on realistic data (perhaps a dump of the staging database)?

I'm worried that, for large volumes of data, the performance of the subquery will be poor and we might be better off using appropriate GROUP BY / JOINS to achieve the desired result.

Reply:
I have tested both on our dev and staging environment. For staging alone which have around 40K data, the migration only take 1 second.